pkg/util/globalerror/flags.go

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package globalerror
+
+const (
+	MaxRecvMsgSizeFlag = "distributor.max-recv-msg-size"
```

Comment:
I don't think this belongs here. If it's required for the error message, it should be passed as an argument to the format method.

Reply:
If it's not possible, to do that a little copy/paste or removing the flag from the error message seems like a better choice.